src/OpenLoco/src/Windows/Construction/Construction.h

```diff
@@ -13,20 +13,20 @@ using namespace OpenLoco::World::TileManager;
 
 namespace OpenLoco::Ui::Windows::Construction
 {
-    enum class GhostFlags : uint8_t
+    enum class GhostVisibilityFlags : uint8_t
     {
         none = 0U,
-        constructArrowVisible = 1U << 0,
-        trackGhostVisible = 1U << 1,
-        signalGhostVisible = 1U << 2,
-        stationGhostVisible = 1U << 3,
-        overheadGhostVisible = 1U << 4,
+        constructArrow = 1U << 0,
+        track = 1U << 1,
+        signal = 1U << 2,
+        station = 1U << 3,
+        overhead = 1U << 4,
     };
-    OPENLOCO_ENABLE_ENUM_OPERATORS(GhostFlags);
+    OPENLOCO_ENABLE_ENUM_OPERATORS(GhostVisibilityFlags);
 
     static loco_global<uint16_t[44], 0x004F891C> _trackPieceToFlags; // Roll into TrackData?
     static loco_global<uint8_t, 0x00522095> _byte_522095;
-    static loco_global<GhostFlags, 0x00522096> _byte_522096;
+    static loco_global<GhostVisibilityFlags, 0x00522096> _byte_522096;
```

Comment:
We should probably name `_byte_522096` while we're at it. Perhaps just `_ghostVisibilityFlags`?